examples/gitserver/hooks/detect-language

```diff
@@ -14,6 +14,11 @@ set -o pipefail
 function has {
   [[ -n $(git ls-tree --full-name --name-only HEAD ${@:1}) ]]
 }
+function hasglob {
+  git ls-tree --full-name --name-only HEAD | while read NAME; do
+    [[ "$NAME" == $1 ]] && return 0
```

Comment:
yeah i tested w/o the [[ ]] which is why it didn't work for me.  looks good, thanks.